### PR TITLE
Theme date formatting in config file

### PR DESCRIPTION
**Pull request by jordibares**
Please double check the **git-footer.html** as this is one I have not managed to abstract. I am quite new to this so...

Thanks, I hope it makes sense.

### CODE REVIEW

**Comment by alex-shpak** [comment]
Thank you for contribution!

File: README.md [hunk 1]
```diff
@@ -84,6 +84,9 @@ enableGitInfo: true
 disableKinds: ["taxonomy", "taxonomyTerm"]
 
 params:
+  # Configure the date format utilised on the pages - TODO, review git-footer.html
+  DateForm: "January 2, 2006"
```
Comment (alex-shpak) [inline README.md]: I would probably change this to `DateFormat` can't see specific reason to shorten name.

File: layouts/partials/docs/git-footer.html [hunk 2]
```diff
@@ -2,8 +2,8 @@
 <div class="align-center book-git-footer {{ if not .GitInfo }}justify-end{{ else }}justify-between{{ end }}">
   {{ with .GitInfo }}
   <div>
-    <a href="{{ $.Site.Params.BookRepo }}/commit/{{ .Hash }}" title='Last modified {{ .AuthorDate.Local.Format "January 2, 2006 15:04 MST" }} by {{ .AuthorName }}' target="_blank" rel="noopener">
-      <img src="{{ "svg/code-merge.svg" | relURL }}" /> {{ .AuthorDate.Local.Format "Last Modified Jan 2, 2006" }}
+    <a href="{{ $.Site.Params.BookRepo }}/commit/{{ .Hash }}" title='Last modified {{ .AuthorDate.Local.Format "2 January 2 2006 15:04 MST" }} by {{ .AuthorName }}' target="_blank" rel="noopener">
```
Comment (alex-shpak) [inline layouts/partials/docs/git-footer.html]: I guess your intention was to use `.Site.Params.DateForm` here? (same for next line)

**Comment by alex-shpak** [comment]
Also would be good to provide default value if setting is not set.
Like here: https://github.com/alex-shpak/hugo-book/blob/master/layouts/partials/docs/menu-filetree.html#L40

File: README.md (hunk 1 above)
Comment (jordibares) [reply README.md]: Fantastic, thanks

File: layouts/partials/docs/git-footer.html (hunk 2 above)
Comment (jordibares) [reply layouts/partials/docs/git-footer.html]: Yes, still a newby...

**Comment by jordibares** [comment]
Sorry for the simple question but I need a bit of help here... I have done the fixes and uploaded to Github the theme.
Do I have to do another pull request?
Thanks

**Comment by alex-shpak** [comment]
No, it's already here, see [Commits](/alex-shpak/hugo-book/pull/34/commits)

**Comment by alex-shpak** [comment]
You forgot to change `DateForm` variable in config :)

**Comment by jordibares** [comment]
Thank you Alex, done.

**Comment by alex-shpak** [comment]
👍

**Comment by alex-shpak** [comment]
FYI, I will rename `DateFormat` to `BookDateFormat` to be specific that this variable is for theme

**Pull request merged**